woocommerce/woocommerce-ios
# Simple Payments: Add Payments Link Builder

# Why

This PR just adds a simple helper that builds a payment link based on the following format.

```
{:store_address}/checkout/order-pay/{:order_id}/?pay_for_order=true&key={:order_key}
```

PS: This does not have any user-facing changes, that will come in a next PR.

# Testing Steps

- Start the Simple Payments flow
- Go all the way until the payment methods screen
- See the payment link being printed in the console
- Copy the link in a browser and you should be able to pay for the order if you have some payment method enabled in your store.


---
